controllers/foodsController.js

```diff
@@ -1,22 +1,25 @@
+const fetch = require('node-fetch')
 const Food = require('../models').Food
 const pry = require('pryjs')
 
-const add = (req, res) => {
-  let food = req.body.food_name
-      food = food.toLowerCase()
-      food = food[0].toUpperCase() + food.substring(1)
-  Food.create({
-    name: food,
-    calories: req.body.calories
-  })
-  .then(food => {
+const add = async (req, res) => {
+  try {
+    eval(pry.it)
+    let food = sanitizeEntry(req.query.food_name)
+    food = await Food.create({
+      name: food,
+      calories: req.query.calories
```

Comment:
Did you figure out how to get this working not just with Postman, but from other browsers too?

Reply:
I had to change a CORS setting on the recipe service... we might have to do that to the calorie app too...